components/library/libwebp/libwebp.p5m

```diff
@@ -1,40 +1,121 @@
+file path=usr/lib/$(MACH64)/libwebp.so.6.0.1
+file path=usr/lib/$(MACH64)/pkgconfig/libwebp-0.5.1.pc
+
+link path=usr/lib/libwebp.so target=libwebp.so.6.0.1
+link path=usr/lib/libwebp.so.6 target=libwebp.so.6.0.1
+file path=usr/lib/libwebp.so.6.0.1
+file path=usr/lib/pkgconfig/libwebp-0.5.1.pc
+
+link path=usr/lib/$(MACH64)/libwebpdecoder.so target=libwebpdecoder.so.2.0.1
+link path=usr/lib/$(MACH64)/libwebpdecoder.so.2 target=libwebpdecoder.so.2.0.1
+file path=usr/lib/$(MACH64)/libwebpdecoder.so.2.0.1
+file path=usr/lib/$(MACH64)/pkgconfig/libwebpdecoder.pc
+
+link path=usr/lib/libwebpdecoder.so target=libwebpdecoder.so.2.0.1
+link path=usr/lib/libwebpdecoder.so.2 target=libwebpdecoder.so.2.0.1
+file path=usr/lib/libwebpdecoder.so.2.0.1
+file path=usr/lib/pkgconfig/libwebpdecoder.pc
+
+
+link path=usr/lib/$(MACH64)/libwebpdemux.so target=libwebpdemux.so.2.0.0
+link path=usr/lib/$(MACH64)/libwebpdemux.so.2 target=libwebpdemux.so.2.0.0
+file path=usr/lib/$(MACH64)/libwebpdemux.so.2.0.0
+file path=usr/lib/$(MACH64)/pkgconfig/libwebpdemux.pc
+
+link path=usr/lib/libwebpdemux.so target=libwebpdemux.so.2.0.0
+link path=usr/lib/libwebpdemux.so.2 target=libwebpdemux.so.2.0.0
+file path=usr/lib/libwebpdemux.so.2.0.0
+file path=usr/lib/pkgconfig/libwebpdemux.pc
+
+
+link path=usr/lib/$(MACH64)/libwebpmux.so target=libwebpmux.so.2.0.1
+link path=usr/lib/$(MACH64)/libwebpmux.so.2 target=libwebpmux.so.2.0.1
+file path=usr/lib/$(MACH64)/libwebpmux.so.2.0.1
+file path=usr/lib/$(MACH64)/pkgconfig/libwebpmux.pc
+
+link path=usr/lib/libwebpmux.so target=libwebpmux.so.2.0.1
+link path=usr/lib/libwebpmux.so.2 target=libwebpmux.so.2.0.1
+file path=usr/lib/libwebpmux.so.2.0.1
+file path=usr/lib/pkgconfig/libwebpmux.pc
+
+
+link path=usr/lib/$(MACH64)/libwebpextras.so target=libwebpextras.so.0.0.0
+link path=usr/lib/$(MACH64)/libwebpextras.so.0 target=libwebpextras.so.0.0.0
+file path=usr/lib/$(MACH64)/libwebpextras.so.0.0.0
+file path=usr/lib/$(MACH64)/pkgconfig/libwebpextras.pc
+
+link path=usr/lib/libwebpextras.so target=libwebpextras.so.0.0.0
+link path=usr/lib/libwebpextras.so.0 target=libwebpextras.so.0.0.0
+file path=usr/lib/libwebpextras.so.0.0.0
+file path=usr/lib/pkgconfig/libwebpextras.pc
+
+
+# Older version is ABI-incompatible; we provide its libs (only built libwebp)
 link path=usr/lib/$(MACH64)/libwebp.so.5 target=libwebp.so.5.0.0
 file path=usr/lib/$(MACH64)/libwebp.so.5.0.0
-file path=usr/lib/$(MACH64)/pkgconfig/libwebp.pc
-link path=usr/lib/libwebp.so target=libwebp.so.5.0.0
+
 link path=usr/lib/libwebp.so.5 target=libwebp.so.5.0.0
 file path=usr/lib/libwebp.so.5.0.0
-file path=usr/lib/pkgconfig/libwebp.pc
-file path=usr/share/man/man1/cwebp.1
-file path=usr/share/man/man1/dwebp.1
+
+file path=usr/lib/$(MACH64)/pkgconfig/libwebp-0.4.0.pc
+file path=usr/lib/pkgconfig/libwebp-0.4.0.pc
+
+
+# Include files are also a bit different for two releases, so we provide both
+file path=usr/include/webp-0.4.0/decode.h
```

Comment:
I was wrong, the headers are pretty compatible. One notable difference is in a structure of YUV colorspaces supported. Also many earlier reserved+padded entries are now renamed and used apparently - but in a size-compatible manner.